app/models/hourly_temp.rb

```diff
@@ -0,0 +1,7 @@
+class HourlyTemp
+
+  def initialize(darksky_api_data, index)
+    @time = Time.at(darksky_api_data[:hourly][:data][index][:time])
+    @temperature = darksky_api_data[:hourly][:data][index][:temperature]
+  end
```

Comment:
Based on my comment above, if you are only passing in one day's worth of data into this poro this could change to:

def initiailize(hourly_data)
  @time = Time.at(hourly_data[:time])
  @temperature = hourly_data[:temperature]
end